relnotes/reverse.feature.md

```diff
@@ -0,0 +1,5 @@
+* `ocean.core.array.Iteration`
+
+  New module containing single function for now - `reverse`, which allows to
+  iterate given slice in reverse order without any allocations (similar to
+  built-in `foreach_reverse`).
```

Comment:
I can't understand why you are so eager to use badly designed utility simply because it is built-in into the language. If it was just another library function which worked like that, I doubt anyone would object deprecating it and replacing with a more reliable version. How is language builti-in any better?